### DISCUSSION TITLE
GetActiveModel

### DISCUSSION THREAD

**Discussion by jlemeriel** [Dlubal-Software/RFEM_Python_Client#372] [Q&A] [answered]
Hi there, 

I would like to interact with an active model, without having to specify  the name of the model. 

For now, in the initModel, you have to put the model_name like this: 
 Model(False,"Bloc 6 RFEM6.rf6") 

Is somebody have the solution ?  

In RFEM /VBA it works perfectly. 
 Dim iMod As RFEM5.IModel2
    Set iMod = iApp.GetActiveModel

**Comment by heetrojivadiya** [comment, marked as answer]
Hello @jlemeriel ,

You can access active model in python by simply calling model with False and empty string. For example : `Model(False, '')`.

Let us know if it works for you or not.

**Comment by jlemeriel** [comment]
Hi
No unfortunately it doesn't work for me . 
![Bug Model](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/167471486/f735e788-b006-46c6-880a-58d40e0b364f)

**Reply by heetrojivadiya** [reply]
Hey,

It should be worked. Try to update RFEM package and run again. If it won't work for you after update then please share code and model for better inspection.

**Reply by jlemeriel** [reply]
Hello heetrojivadiya !
Indeed, it works with the last version of RFEM package.  

Thanks again
